<?xml version="1.0"?>
<launch>
<node name="hector_barrel_detection_node" pkg="hector_barrel_detection" type="hector_barrel_detection_node" respawn="true" output="screen">
<param name="r_min" type="int" value="0" />
<param name="r_max" type="int" value="20" />
<param name="g_min" type="int" value="0" />
<param name="g_max" type="int" value="20" />
<param name="b_min" type="int" value="10" />
<param name="b_max" type="int" value="255" />
<param name="percentage_blue" type="int" value="8" />
</node>

</launch>
